src/test/java/rx/SingleTest.java

```diff
@@ -2005,4 +2005,32 @@ public void unsubscribeComposesThroughAsync() {
         assertFalse("Observers present?!", ps.hasObservers());
     }
 
+    @Test
+    public void flatMapCompletableComplete() {
+        TestSubscriber testSubscriber = new TestSubscriber();
+
+        Single.just(1).flatMapCompletable(new Func1<Integer, Completable>() {
+            @Override
+            public Completable call(final Integer integer) {
+                return Completable.complete();
```

Comment:
Would be better to test against more complicated `Completable`, for example check that some function was invoked